R/lazyframe__lazy.R

```diff
@@ -213,6 +213,18 @@ LazyFrame_limit = function(n) {
   unwrap(.pr$LazyFrame$limit(self,n))
 }
 
+#' @title Tail
+#' @description take last n rows of query
+#' @keywords LazyFrame
+#' @param n positive numeric or integer number not larger than 2^32
+#'
+#' @details any number will converted to u32. Negative raises error
+#'
+#' @return A new `LazyFrame` object with applied filter.
+LazyFrame_tail = function(n) {
+  if(!is.numeric(n)) stopf("tail: n must be numeric")
```

Comment:
most R side type checks becomes redundant when using robj_to! and can just be dropped.

Reply:
Done